src/client/haproxy.py

```diff
@@ -45,22 +55,26 @@ def checkBackendConf(self):
 
     def backendConfReady(self, stat):
         stat = stat.split("\n")
-        backendExists = False
+        backendsExist = False
+        existingBackends = 0
         fallbackBackendExists = False
         for backend in stat:
             values = backend.split(",")
             ## Example backend line
             ## http_back,mywebapp2,0,0,0,0,,0,0,0,,0,,0,0,0,0,UP,10,1,0,0,1,330587,0,,1,4,2,,0,,2,0,,0,L4OK,,0,0,0,0,0,0,0,,,,,0,0,,,,,-1,,,0,0,0,0,,,,Layer4 check passed,,2,3,4,,,,10.14.34.198:80,,http,,,,,,,,0,0,0,,,0,,0,0,0,0,
-            if len(values) > 80 and values[0] == self.backendName and values[1].startswith(self.backendBaseName):
-                backendExists = True
-            elif self.azLimiter == "true" and len(values) > 80 and values[0] == self.fallbackBackendName and values[1].startswith(self.fallbackBackendBaseName):
-                fallbackBackendExists = True
+            for backend in backendList:
+                if len(values) > 80 and values[0] == backend and values[1].startswith(backendList[backend]["baseName"]):
```

Comment:
```suggestion
            for k,v in backendList.items():
                if len(values) > 80 and values[0] == k and values[1].startswith(v["baseName"]):
```
you can iterate over a dictionnary with `.items()`